src/components/playback/mediasession.js

```diff
@@ -241,6 +241,15 @@ import connectionManager from 'connectionManager';
         navigator.mediaSession.setActionHandler('seekforward', function () {
             execute('fastForward');
         });
+
+        /* eslint-disable-next-line compat/compat */
+        navigator.mediaSession.setActionHandler('seekto', function (object) {
+            let item = playbackManager.getPlayerState(playbackManager.getCurrentPlayer()).NowPlayingItem;
```

Comment:
Why not use currentPlayer instead of playbackManager.getCurrentPlayer()?

Reply:
yes should be currentPlayer